src/modm/platform/clock/sam_pmc/clockgen.hpp.in

```diff
@@ -105,16 +205,31 @@ class ClockGen {
 	 */
 	static void enableExternal32Khz(bool crystal_bypass = false);
 
-	template<uint32_t multiplier>
+	template <MainInternalFreq freq>
+	static void enableMainInternal();
+
+	template <bool bypass>
+	static void enableMainExtermal();
```

Comment:
```suggestion
	static void enableMainExternal();
```

Why is this a template though? I guess because the other functions are templates?

Reply:
This one is technically a missing symbol, I didn't implement it (don't have HW with a crystal yet) and just left out a stub for the next person or myself when I do get the next revision with a crystal. And yes, I just assumed it will take less resources as a template.

Reply:
Fixed (both argument and a typo).